controllers/yawol-controller/loadbalancer/loadbalancer_controller.go

```diff
@@ -37,8 +37,11 @@ import (
 )
 
 const (
-	DefaultRequeueTime = 10 * time.Millisecond
-	ServiceFinalizer   = "yawol.stackit.cloud/controller2"
+	DefaultRequeueTime          = 10 * time.Millisecond
+	ServiceFinalizer            = "yawol.stackit.cloud/controller2"
+	KeepFloatingIPAnnotation    = "yawol.stackit.cloud/keepFloatingIP"
+	KeepPortAnnotation          = "yawol.stackit.cloud/keepPort"
+	KeepSecurityGroupAnnotation = "yawol.stackit.cloud/keepSecurityGroup"
```

Comment:
Can you move this to the other annotations for loadbalancer and use the same saming?: 

https://github.com/stackitcloud/yawol/blob/19dcae4f7bee3a8f41275e26e1f0220fcfb8885e/api/v1beta1/loadbalancer_types.go#L62-L72

Reply:
done